.github/workflows/test.yml

```diff
@@ -10,6 +10,15 @@ jobs:
   test:
     name: Test
     runs-on: ubuntu-latest
+    services:
```

Comment:
Damn, pretty easy to run IPFS in CI like this.

Reply:
Totally, maybe it just used compose under the hood, or at least puts all services on the same network. @carsonfarmer you might try this regarding your questions yesterday. You could probably run two `go-threads` services this way.